src/models/entities/UserGroup.ts

```diff
@@ -1,58 +1,164 @@
+import { group } from 'console';
 import { IUserGroup } from 'src/interfaces/IUserGroup';
 import { Field, ID, ObjectType } from 'type-graphql';
 import { BaseEntity, Column, Entity, JoinTable, ManyToMany, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
 import { Challenge, Ecoverse, Tag, User } from '.';
+import { IGroupable } from '../interfaces';
```

Comment:
It might be best to split the model interfaces from the GraphQL schema interfaces